vhdbuilder/packer/components.json

```diff
@@ -125,7 +125,8 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "ciprod08102022",
```

Comment:
remove old versions and keep only new version.

Reply:
removed